dummy-api/matrix-service-api.yaml

```diff
@@ -0,0 +1,115 @@
+swagger: '2.0'
+info:
+  title: HCA Matrix Service
+  version: 0.0.0-dummy
+schemes:
+  - http
```

Comment:
Based on https://github.com/HumanCellAtlas/dcp/issues/102 - I don't think that the http scheme should be supported.

Reply:
👍 yeah want to follow whatever the dcp's doing for this.